### ISSUE TITLE
Move statement implemention

### ISSUE THREAD

**Issue by pravdomil** [klazuka/intellij-elm#615]
Move statement actions works just like Move line, is it possible to make work so that it reorder statements?

<img width="330" alt="image" src="https://user-images.githubusercontent.com/2387356/73886392-cc9f9d80-4869-11ea-85e7-fb747a1abd07.png">

**Comment by klazuka** [comment]
Yes, this can and should be implemented eventually. I don't think it would be that hard to do.
